.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "baselayer"]
 	path = baselayer
 	url = git://github.com/cesium-ml/baselayer.git
-	branch = master
+	branch = firefox
```

Comment:
Should we merge this into master before continuing?

Reply:
Sounds good - wanna merge my new baselayer PR?

Reply:
I see those tests are failing; do you know why?

Reply:
@stefanv yeah, because those changes break the cesium_web tests, and the fixes are in this PR. So they'll need to be merged simultaneously

Reply:
Actually we'll need to merge this first, then baselayer, then update this to use the master branch.